discovery-handlers/onvif/src/discovery_impl.rs

```diff
@@ -352,6 +352,7 @@ pub mod util {
             "simple_onvif_discover - uris after filtering by scopes {:?}",
             filtered_uris
         );
+        filtered_uris.dedup();
```

Comment:
Would it make sense to have broadcast_responses to be a set so that we do not capture duplicates in the first place or dedup on broadcast_responses prior to calling get_scope_filtered_uris_from_discovery_response?

Reply:
Good point! Switched to a hashset in 80bcc3b8295272